src/index.tsx

```diff
@@ -1482,6 +1567,231 @@ class App extends React.Component<{}, AppState> {
+                      className="swatch-input"
+                      value={
+                        getSelectedStrokeColor() ||
+                        this.state.currentItemStrokeColor
+                      }
+                      onChange={e => this.changeSrokeColor(e.target.value)}
+                    />
+                  </div>
+                </div>
+
+                {someElementIsSelectedIsRectangleOrEllipseOrDiamond() && (
+                  <div className="panelColumn">
+                    <h5>Shape Background Color</h5>
+                    <div>
+                      <button
+                        className="swatch"
+                        style={{
+                          backgroundColor:
+                            getSelectedBackgroundColor() ||
+                            this.state.currentItemBackgroundColor
+                        }}
+                        onClick={() =>
+                          this.setState(s => ({
+                            currentColorPicker:
+                              s.currentColorPicker ===
+                              ColorPicker.SHAPE_BACKGROUND
+                                ? null
+                                : ColorPicker.SHAPE_BACKGROUND
+                          }))
+                        }
+                      />
+                      {this.state.currentColorPicker ===
+                      ColorPicker.SHAPE_BACKGROUND ? (
+                        <div className="popover">
+                          <div
+                            className="cover"
+                            onClick={() =>
+                              this.setState({ currentColorPicker: null })
+                            }
+                          />
+                          <SketchPicker
+                            color={this.state.currentItemBackgroundColor}
+                            onChange={color =>
+                              this.changeBackgroundColor(color.hex)
+                            }
+                          />
+                        </div>
+                      ) : null}
+                      <input
+                        type="text"
+                        className="swatch-input"
+                        value={
+                          getSelectedBackgroundColor() ||
+                          this.state.currentItemBackgroundColor
+                        }
+                        onChange={e =>
+                          this.changeBackgroundColor(e.target.value)
+                        }
+                      />
+                    </div>
+                  </div>
+                )}
+              </>
+
+              {someElementIsSelectedIsRectangleOrEllipseOrDiamond() && (
+                <>
+                  <h4>Fill</h4>
+                  <div className="panelColumn">
+                    {/* <select onChange={this.changeFillStyle} value={getSelectedFillStyles()}> */}
+                    <button
```

Comment:
Use select input here

Reply:
@vjeux what do you think? Should we use a select or a list of options?